### DISCUSSION TITLE
How to pull down the "submodel"  resources

### DISCUSSION THREAD

**Discussion by masterchen** [CCXXXI/ecg_monitor#344] [Q&A] [answered]
I got  an error  say,  cpp files missing  while  build the resource , then i checked the directory of the ios/calsses  , the folder is empty .
info as follow :

  from D:\OpenSource\github\Flutter\ecg_monitor\android\app
  CMake Error at CMakeLists.txt:9 (add_library):
    Cannot find source file:

      ../../ios/Classes/PanTompkinsQRS/src/panTompkins.cpp

    Tried extensions .c .C .c++ .cc .cpp .cxx .cu .mpp .m .M .mm .ixx .cppm .h
    .hh .h++ .hm .hpp .hxx .in .txx .f .F .for .f77 .f90 .f95 .f03 .hip .ispc

**Comment by CCXXXI** [comment, marked as answer]
See: https://git-scm.com/book/en/v2/Git-Tools-Submodules
TL;DR: Add `--recurse-submodules` to `git clone` or run `git submodule update --init --recursive` after clone.
